index.html

```diff
@@ -1,14 +1,108 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Mockup Home</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Mockup Home</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+</head>
+
+<body>
+
+
+
+  <h1>Testing</h1>
+
+  <div class="allPicsDiv">
+    <div class="picDiv"><img src=http://placekitten.com/400/200> Text</div>
+    <div class="picDiv"><img src=http://placekitten.com/400/200> Text</div>
+    <div class="picDiv"><img src=http://placekitten.com/400/200> Text</div>
+  </div>
+
+  <h1>Tell me more</h1>
+  <div class="quoteBanner">
+    This is a great quote
+
+  </div>
+  <div class="bothTextCol">
+
+    <div class="leftTextCol">
+      Something about aliens
+      <img src=http://placekitten.com/800/400>
+    </div>
+
+    <div class="rightTextCol">
+      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Accusamus dolore quas nam sint ea, eveniet velit illum
+      placeat officia numquam odit repellat alias ut ratione itaque neque. Obcaecati, facere animi! Explicabo neque sint
+      officiis aperiam dolore minima cumque libero quos, maiores eos, impedit nam, provident praesentium. Eaque delectus
+      asperiores dolorum nulla necessitatibus, hic sapiente aspernatur temporibus perspiciatis repudiandae non nesciunt
+      enim quod reiciendis esse dolor quo quis ratione ad distinctio perferendis ipsa. Tenetur deleniti numquam et
+      quasi. Placeat est, expedita ab dolore commodi reiciendis perspiciatis consectetur assumenda rem! Reprehenderit
+      repudiandae quam ratione dolore consequuntur repellat ipsum sunt fugit. Quia dolore eaque magni perspiciatis nam
+      tenetur quis officiis! Corrupti velit ducimus facilis! Iste hic distinctio animi enim ipsum consequuntur
+      doloremque, neque temporibus assumenda aliquid ratione quam quis delectus at. Quibusdam id eos, enim dolorum atque
+      provident delectus ratione aperiam beatae. Corrupti nobis excepturi animi nesciunt voluptas culpa at, molestiae
+      esse dolore, quasi libero illum a non, error architecto dicta id minima! Nostrum aliquid quaerat consequuntur,
+      velit illo ullam sapiente laboriosam ut ab, doloribus dicta aliquam distinctio consectetur, officia natus
+      expedita. Perferendis doloremque saepe veritatis placeat facilis, beatae harum a error aut dolore sapiente
+      suscipit corporis ea inventore sit quis. Reiciendis, non!

```

Comment:
Text tags will make your code easier to read, both for people, and for screen readers. It will also make it easier to style your code.